__docs__/wonder-blocks-form/text-field.stories.tsx

```diff
@@ -624,6 +629,30 @@ Disabled.parameters = {
     },
 };
 
+/**
+ * A TextField can be used with custom styles. The following parts can be styled:
+ * - `root`: Styles the root element
+ * - `input`: Styles the underlying `input` element
+ */
+export const CustomStyles: StoryComponentType = {
+    args: {
+        value: "Value",
+        styles: {
+            root: {
+                backgroundColor:
+                    semanticColor.core.background.instructive.subtle,
+                padding: sizing.size_140,
+            },
+            input: {
+                backgroundColor: semanticColor.surface.secondary,
+                border: `${border.width.thin} solid ${semanticColor.core.border.instructive.default}`,
+                color: semanticColor.core.foreground.instructive.strong,
+            },
+        },
+    },
+};
+
+// TODO(WB-2004): Remove this story once `style` prop is removed.
```

Comment:
Related: I created WB-2004 to remove the `style` prop from `TextField`. `TextArea` has `style` and `rootStyle` props that can be consolidated into `styles` in WB-2005